dev/php.ini

```diff
@@ -1,17 +1,28 @@
 [PHP]
-; Prod settings -- see ../prod/php.ini
```

Comment:
What's the reason for removing the reference to `../prod/php.ini`? The idea was that the sections are kept in sync.